back/engines/commercial/user_custom_fields/spec/acceptance/stats_users_spec.rb

```diff
@@ -143,7 +143,7 @@ def xlsx_worksheet_to_array(worksheet)
     end
   end
 
-  describe 'by_domicile endpoints' do
+  describe 'former by_domicile endpoints' do
```

Comment:
I have the same comment as for the gender PR: https://github.com/CitizenLabDotCo/citizenlab/pull/7664#discussion_r1585555499.